js/components/Apple2.tsx

```diff
@@ -38,12 +39,13 @@ export interface Apple2Props {
  * @returns
  */
 export const Apple2 = (props: Apple2Props) => {
-    const { e, sectors } = props;
+    const { e, enhanced, sectors } = props;
     const screen = useRef<HTMLCanvasElement>(null);
     const [apple2, setApple2] = useState<Apple2Impl>();
     const [io, setIO] = useState<Apple2IO>();
     const [cpu, setCPU] = useState<CPU6502>();
     const [error, setError] = useState<unknown>();
+    const drivesReady = useMemo(() => new Ready(), []);
```

Comment:
Below I mention adding a constructor parameter to `Ready` for the reject callback. If you did that, this could be something like:

```suggestion
    const drivesReady = useMemo(() => new Ready(setError), []);
```